.kuzzlerc.sample

```diff
@@ -257,6 +257,11 @@
       "proxy": false
     },
 
+    // [vaultSeed]
+    // Initialization vector used to encrypt secrets
+    // Random seed can be generated with: bin/kuzzle vaultSeed
+    "vaultSeed": "764c1afa2988981a",
```

Comment:
Isn't that property outdated now that we create a new seed for each encrypted value?
(same for the one in the default.config.js file)